spec/features/admin/omb_cx_reporting_collections_spec.rb

```diff
@@ -57,6 +57,20 @@
         expect(page).to have_content("Omb cx reporting collection was successfully created.")
       end
     end
+
+    describe "heartbeat" do
+      xit "display successful flash message after more than 15 mins on page" do
```

Comment:
I think https://api.rubyonrails.org/v5.2.6/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel could help here.

I used to use [delorean](https://github.com/bebanjo/delorean), but it is deprecated.